services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/VpcTopicCache.java

```diff
@@ -77,4 +77,16 @@ public void addSubscribedNodeForVpc(String vpcId, String topic, String hostIp, S
             tx.commit();
         }
     }
+
+    @DurationStatistics
+    public void deleteNodeSubscribeInfo(String vpcId, String hostIp) throws Exception {
+        try (Transaction tx = vpcTopicInfoICache.getTransaction().start()) {
+            VpcTopicInfo vpcTopicInfo = vpcTopicInfoICache.get(vpcId);
+            if (vpcTopicInfo != null) {
+                vpcTopicInfo.getSubscribeMapping().remove(hostIp);
+            }
+            vpcTopicInfoICache.put(vpcId, vpcTopicInfo);
+            tx.commit();
```

Comment:
@VanderChen So we allow to put a `null` into `vpcTopicInfoICache` for certain `vpcId`?